extraTs/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+interface Permission {
```

Comment:
Each interfaces should start with 'I'
e.g.    Permission can be defined as IPermission

Reply:
Done changes